### PR TITLE
updated webpack

### PR DESCRIPTION
**Pull request by ardianck**
Added latest changes to webpack

### CODE REVIEW

**Comment by ardianck** [comment]
adding max length feature

File: react-validated/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (visar-polymath) [inline react-validated/package-lock.json]: @ardianck what is this file for?

**Review by visar-polymath — APPROVED** [review]

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+webpack.config.js
```
[the remark this answers is not in the document]
Comment (ardianck) [reply .npmignore]: @visar-polymath yes, we don't need it. It can be removed.

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
```
[the remark this answers is not in the document]
Comment (ardianck) [reply src/App.js]: @visar-polymath it was left behind from my previous setup.

File: react-validated/package-lock.json (hunk 1 above)
Comment (ardianck) [reply react-validated/package-lock.json]: @visar-polymath this is generated by package.json. Maybe I did not get your question?

Comment (visar-polymath) [reply react-validated/package-lock.json]: @ardianck note the path, it was a bad artifact... I removed that whole folder

File: src/App.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (visar-polymath) [reply src/App.js]: @ardianck let's just be cautious with these in the future... undoing each other's work is counter-productive

[the remark this answers is not in the document]
Comment (ardianck) [reply src/App.js]: ok, I'll try to be more cautious in the future.

**Pull request merged**